src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -105,13 +116,26 @@ void* LoadCoreClr(std::string& runtimeDirectory)
         runtimeDirectory = coreClrEnvVar;
 
         std::string coreClrDllPath = runtimeDirectory;
```

Comment:
(This is a general note - I wonder if we need a general try/catch(const &std:exception) at the entry point somewhere - these could potentially throw `std::bad_alloc`  which will lead to a crash if not handled somehow)